R/nepal_sees_jags_post.R

```diff
@@ -0,0 +1,19 @@
+#'
+#'
+#' SEES Typhoid run_mod jags output
+#'
+#' An [serodynamics::run_mod()] output using sees_nepal data as input, 
+#' stratifying by bldculres, which is the diagnosis type (typhoid or
+#' paratyphoid).
+#'
+#' @format ## `nepal_sees_jags_post`
+#' A jags.post [list] object or multiple jags.post [list]
+#' if stratified. Returned as a [list] of class [runjags::runjags-class]
+#' \describe{
+#'  \item{curve_params}{A [data.frame] titled `curve_params` that contains the
+#'   posterior distribution}
+#'  \item{jags.post}{An object with class [runjags::runjags]}
```

Comment:
```suggestion
#'  \item{jags.post}{A [list] of [runjags::runjags-class] objects}
```
if I'm understanding correctly:
![image](https://github.com/user-attachments/assets/ff43b71a-9bce-4204-8672-4a1c2fb054b6)

`[runjags::runjags]` unfortunately redirects to ``?runjags::`runjags-package` `` instead of ``?runjags::`runjags-class` ``

Fortunately, `{roxygen2}` will format the resulting Rd doc file nicely:

![image](https://github.com/user-attachments/assets/8b089c9a-9fb9-402a-9769-b22fc5d53d1d)

Reply:
(meant to formally "request changes" but clicked "comment" instead in the review menu.